.travis/publish-gh-pages.sh

```diff
@@ -3,13 +3,6 @@
 VERSION=`node -e "console.log(require('./package.json').version)"`
 TMP_GH_PAGES_DIR=.gh-pages-demo
 
-# We only want to deploy to gh-pages from "master"
-if [ "${TRAVIS_BRANCH}" != "master" ]
-then
-    echo "Skipping gh-pages publish for branch ${TRAVIS_BRANCH}"
-    exit 0
-fi
-
```

Comment:
Bunsen is doing everything correctly except publishing to gh-pages. That'll be fixed once https://github.com/ciena-frost/ember-frost-bunsen/pull/371/files is merged.